se_denseslam/src/preprocessing.cpp

```diff
@@ -208,9 +215,9 @@ void halfSampleRobustImageKernel(se::Image<float>& out,
 			const float center = in[centerPixel.x() + centerPixel.y() * in.width()];
 			for (int i = -r + 1; i <= r; ++i) {
 				for (int j = -r + 1; j <= r; ++j) {
-          Eigen::Vector2i cur = centerPixel + Eigen::Vector2i(j, i); 
-          se::math::clamp(cur, 
-                Eigen::Vector2i::Constant(0), 
+          Eigen::Vector2i cur = centerPixel + Eigen::Vector2i(j, i);
+          se::math::clamp(cur,
+                Eigen::Vector2i::Constant(0),
```

Comment:
`Eigen::Vector2i::Zero()`?  it is slightly more readable

Reply:
Changed Constant() to Zero()/Ones() where appropriate.